AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -564,15 +564,14 @@ - (void)relayoutAllRows
     LOG(@"Edit Command - relayoutRows");
     [_editingTransactionQueue waitUntilAllOperationsAreFinished];
     
-    NSArray *indexPaths = ASIndexPathsForMultidimensionalArray(_completedNodes);
-    NSArray *loadedNodes = ASFindElementsInMultidimensionalArrayAtIndexPaths(_completedNodes, indexPaths);
-    
-    for (NSUInteger i = 0; i < loadedNodes.count && i < indexPaths.count; i++) {
-      ASSizeRange constrainedSize = [_dataSource dataController:self constrainedSizeForNodeAtIndexPath:indexPaths[i]];
-      ASCellNode *node = loadedNodes[i];
-      [node measureWithSizeRange:constrainedSize];
-      node.frame = CGRectMake(0.0f, 0.0f, node.calculatedSize.width, node.calculatedSize.height);
-    }
+    [_completedNodes enumerateObjectsUsingBlock:^(NSMutableArray *section, NSUInteger sectionIndex, BOOL *stop) {
```

Comment:
While I should profile these versions, I actually didn't :( My main motivation for this diff was about readability/maintainability because of its infrequent usage. Regarding performance, my assumption was that enumerate existing array should be faster than array allocations and finding elements.

I've just done profiling and the new version is actually much faster. I ran -relayoutAllRows 10,000 times on the same constrained size, so cached layouts were used and there was no actual layout calculation. The old version took 610ms. Almost 40% of that time was spent on ASIndexPathsForMultidimensionalArray and ASFindElementsInMultidimensionalArrayAtIndexPaths. The new version took 419ms. And creating NSIndexPaths only took 0.2% of that time. On both versions, one of the most expensive operations is ASDisplayNode's setFrame, which is unavoidable I guess.

![old_version](https://cloud.githubusercontent.com/assets/587874/9311035/717a9c84-451c-11e5-8bad-cdb7b07926d2.png)

![new_version](https://cloud.githubusercontent.com/assets/587874/9311039/753e3254-451c-11e5-82bd-c21b2d8f5ded.png)